src/styles/foundation/layout.scss

```diff
@@ -14,6 +14,9 @@ $layout-width-data: (
     min: rem(240px),
     max: rem(320px),
   ),
+  equal-width: (
```

Comment:
Small nitpick, but this should probably also be `even-width` to match the prop name.